src/test/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerCreationTest.java

```diff
@@ -0,0 +1,265 @@
+                                TapChangerStepCreationInfos.builder()
+                                        .index(3)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .alpha(0.)
+                                        .build()
+                        )).build())
+                .ratioTapChanger(RatioTapChangerCreationInfos.builder()
+                        .lowTapPosition(5)
+                        .tapPosition(6)
+                        .regulating(true)
+                        .targetDeadband(1.)
+                        .regulatingTerminalId("v2load")
+                        .regulatingTerminalVlId("v2")
+                        .regulatingTerminalType("LOAD")
+                        .loadTapChangingCapabilities(true)
+                        .targetV(5.)
+                        .steps(List.of(TapChangerStepCreationInfos.builder()
+                                        .index(5)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build(),
+                                TapChangerStepCreationInfos.builder()
+                                        .index(6)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build(),
+                                TapChangerStepCreationInfos.builder()
+                                        .index(7)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build(),
+                                TapChangerStepCreationInfos.builder()
+                                        .index(8)
+                                        .rho(1)
+                                        .r(0)
+                                        .x(0)
+                                        .g(0)
+                                        .b(0)
+                                        .build()
+                        ))
+                        .build())
+                .build();
+    }
+
+    @Override
+    protected ModificationInfos buildModificationUpdate() {
+        return TwoWindingsTransformerCreationInfos.builder()
+                .type(ModificationType.TWO_WINDINGS_TRANSFORMER_CREATION)
```

Comment:
this update-case DTO is an exact copy/paste from the creation-case DTO.
you must change some values

Reply:
you should change even more data, as mush as you can